tasks/main.yml

```diff
@@ -46,21 +62,40 @@
     state: link
   when: "ansible_service_mgr != 'systemd'"
 
+- name: Set RUN_AS user to sonar, to avoid ElasticSearch failing with 'cannot run as root'
+  lineinfile:
+    path: /usr/local/sonar/bin/linux-x86-64/sonar.sh
+    firstmatch: yes
+    regexp: '^#RUN_AS_USER='
```

Comment:
This doesn't make sense to me, why don't you remove `firstmatch: yes` and instead set regexp to simply `'RUN_AS_USER'` as that should match the line regardless if it's commented or not and replace it with the correct line

Reply:
ok can do. I was trying to figure out how to make ansible report it changed the line once. If you run the playbook, it always says changed but it doesn't actually change it after the first run. I was using firstmatch after reading more on the module.